1-exercises/D-do-while/exercise.js

```diff
@@ -7,7 +7,13 @@
 */
 
 function evenNumbersSum(n) {
-    // TODO
+let sum = 0; 
```

Comment:
Please remember about correct code formatting. It is making your life easier when You are coding and reading code.